tests/test_focal_loss.py

```diff
@@ -132,7 +132,7 @@ def test_consistency_with_cross_entropy_2d_no_reduction(self):
             error = np.abs(a - b)
             max_error = np.maximum(error, max_error)
 
-        assert np.allclose(max_error, 0)
+        assert np.allclose(max_error, 0, atol=1e-6)
```

Comment:
This implies that the Pytorch `BCEWithLogitsLoss` is less accurate now? It might not matter during training but it's not a good thing anyway if this is what's happened.

Reply:
Makes sense here now thanks!